src/model/services/mapi/capiv1alpha3/key.ts

```diff
@@ -1,9 +1,20 @@
-import { ICluster } from './';
+import * as corev1 from 'model/services/mapi/corev1';
+
+import { ICluster, IStatusCondition } from './';
 
 export const labelOrganization = 'giantswarm.io/organization';
 export const labelCluster = 'giantswarm.io/cluster';
 export const labelReleaseVersion = 'release.giantswarm.io/version';
 
+export const conditionTypeCreating = 'Creating';
```

Comment:
The conditions logic is written in a similar way to http://github.com/giantswarm/conditions/